cypress/integration/components/Select.spec.js

```diff
@@ -79,6 +79,76 @@ describe("Select", () => {
     });
   });
 
+  describe("Multiselect with copy/paste CSV functionality", () => {
+    const getCopyCsvLabesButton = () => cy.get("[data-testid='csv-labels']");
+    const getCopyCsvValuesButton = () => cy.get("[data-testid='csv-values']");
+    const getCopyCsvValuesWithDuplicateButton = () => cy.get("[data-testid='csv-labels-with-duplicates']");
+    const selectTextInputSelector = "[data-testid='select-input'] input";
+    const getSelectTextInput = () => cy.get(selectTextInputSelector);
+
+    beforeEach(() => {
+      cy.renderFromStorybook("select--paste-csv-value-in-select");
+    });
+
+    it("copy and paste CSV labels to select", () => {
+      const csvValues = "PCN2, PCN4";
+
+      paste({ destinationSelector: selectTextInputSelector, pasteType: "text/plain", pastePayload: csvValues });
```

Comment:
The default `pasteType` is `text/plain`. Do we need to repeat it here?

Reply:
Yes, we can remove it.